### DISCUSSION TITLE
When Grid rows are collapsed, add value of first column to header to help differentiate rows

### DISCUSSION THREAD

**Discussion by jaredchesebro** [ExpressionEngine/ExpressionEngine#3262] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
When grid rows are collapsed there is no indication of the content within them. See image: 

![grid-rows](https://user-images.githubusercontent.com/21958491/231288265-ba9de1d8-534f-4c84-a10f-fd6127456f1d.jpg)

**Describe the solution you'd like**
I would like to see the value of the first column/field within a grid displayed in the grid header so that it is easier to differentiate between each row when they are collapsed.

**Teachability, Documentation, Adoption, Migration Strategy**
Since this is mostly a UI/UX improvement users would intuitively understand the implementation and benefit immediately.

**Comment by creativearc-paul** [comment]
I second this. Even a “row Id” would help. If an image, show the file name. If a text field, show first several characters.

**Comment by intoeetive** [comment]
Vertical layout is often used when there are several columns in each row... trying to display each of those (even if just few characters) would really make things messy. I agree some sort of preview / heading would be nice, but we really need to think of ideas on how to generate thaty=
